electionleaflets/apps/api/tests/test_feeds.py

```diff
@@ -1,14 +1,51 @@
-from django.test import TestCase
+from django.test import Client
+import pytest
 from django.urls import reverse
+from django.contrib.auth.models import User
+from django.db import connection
 
+from api.feeds import LatestLeafletsFeed
 from leaflets.tests.model_factory import LeafletFactory
 
+@pytest.fixture(autouse=True)
+def create_leaflets():
+    for _ in range(10):
+        LeafletFactory() 
 
-class TestFeeds(TestCase):
-    def setUp(self):
-        # make some leaflets
-        [LeafletFactory() for i in range(10)]
+@pytest.fixture
+def setup_client():
+    client = Client()
+    User.objects.create_user(username='testuser', password='12345')
+    client.login(username='testuser', password='12345')
+    yield client
 
-    def test_latest(self):
-        URL = reverse("latest_feed")
-        self.client.get(URL)
+@pytest.fixture
+def teardown_db():
+    yield
+    with connection.cursor() as cursor:
+        cursor.execute("DELETE FROM leaflets_leaflet")
```

Comment:
Can you explain this, maybe in a comment? This is something we do before deleting the database, so I don't know why we need to clean up one table

Reply:
When I moved from `TestCase` to `pytest`, I didn't see the same output in the terminal that confirms the test database is being used. This can go; I added this when I was unsure whether we needed to tear down each time we ran these tests.

Reply:
I'm also happy to reset these commits and reorganise and/or pull testing commits out into another PR